index.html

```diff
@@ -1,16 +1,44 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <title>Kimia's Portfolio</title>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <script>
       script.js;
     </script>
-    <title>Document</title>
   </head>
   <body>
-    Lorem ipsum dolor sit, amet consectetur adipisicing elit. Dolorum adipisci,
-    itaque ipsam et totam veritatis eum nihil quaerat delectus fuga error maxime
-    placeat consequuntur obcaecati impedit exercitationem illo porro architecto.
+    <h1>Kimia Harris</h1>
+    <img
+      src="software-dev-cpu.jpg"
+      alt="Three computer screens with code"
+      width="260"
+      height="240" />
+    <section id="About">
+      <h2>About</h2>
+      <p>
+        Hi! I'm currently a QC chemist living in Baltimore, MD. I enjoy learning
```

Comment:
Hello, My name is Jie Du, currently a mentor in code the dream, its a pleasure to meet you!!!!!!
I hope you have fun in this course, and hope you do well for the upcoming assignments!